src/qtiCommonRenderer/renderers/interactions/SliderInteraction.js

```diff
@@ -44,6 +44,7 @@ var _slideTo = function(options) {
 const render = function(interaction) {
     const attributes = interaction.getAttributes(),
         $container = interaction.getContainer(),
+        direction = window.getComputedStyle($container[0]).getPropertyValue('direction') || 'ltr',
```

Comment:
**suggestion for the future** 
@jsconan / @krampstudio 
here is the same issue that I had with Liquid PCI, changing language in Authoring will not reflect on the component. 
Authoring is not in the scope of this task but in the future it worth having generic utils/helpers to handle change and rerender the component.

Reply:
The approach used by now was for the authoring to render the item. Isn't it the case already? From a runtime perspective, it's acceptable.

Reply:
> **suggestion for the future** @jsconan / @krampstudio here is the same issue that I had with Liquid PCI, changing language in Authoring will not reflect on the component. Authoring is not in the scope of this task but in the future it worth having generic utils/helpers to handle change and rerender the component.

Do you mean the item component here? Direction is applied on the item component as well